packages/framework/attributor/src/stringInterner.ts

```diff
@@ -61,7 +61,7 @@ export class MutableStringInterner implements StringInterner {
 	 */
 	public getString(internId: number): string {
 		const result = this.internedStrings[internId];
-		if (!result) {
+		if (result === undefined) {
```

Comment:
@Abe27342 Double-checking: I think this change matches the code's intent, but just to make sure: this wasn't depending on `""` being falsy right?

Reply:
it certainly didn't _want_ to depend on that. Seems like an edge case bug fix to me. nice :)